src/controller.ts

```diff
@@ -11,7 +11,7 @@ import { RunHandler, TestRunner } from "./runner";
 import { ISourceMapMaintainer, SourceMapStore } from "./source-map-store";
 
 const diagnosticCollection = vscode.languages.createDiagnosticCollection("nodejs-testing-dupes");
-const jsExtensions = ".{mjs,cjs,js}";
+const jsExtensions = ".{mjs,cjs,js,ts,tsx,mts}";
```

Comment:
This should be user-configured, since these cannot get run out of the box. Perhaps a mapping of extensions to arguments used to run them, so a user could have something like

```js
"nodejs-testing.extensions": [{
  "extensions": ["ts", "tsx", "mts"],
  "parameters": ["--loader", "tsx"]
}]
```

This can then be documented in the readme

Reply:
I added two more commits. The extensions are now configurable along with their associated parameters. The default is defined in the code as follows:

````
"nodejs-testing.extensions": 
[        
        {
            "extensions": [
                "mjs",
                "cjs",
                "js"
            ],
            "parameters": []
        }
]
````

If you have TypeScript unit tests and JS unit tests, the configuration should be done as follows:

````
 "nodejs-testing.extensions": [
        {
            "extensions": [
                "mts",
                "cts",
                "ts"
            ],
            "parameters": ["--loader", "tsx"]
        },
        
        {
            "extensions": [
                "mjs",
                "cjs",
                "js"
            ],
            "parameters": []
        }
    ]
````